drivers/video/sunxi/sunxi_display.c

```diff
@@ -2368,10 +2368,11 @@ void *video_hw_init(void)
   lcd_init();
 
   uint16_t bug=3;
+  run_command("load mmc 0:1 0x80000000 miyoo-boot.bmp", 0);
```

Comment:
some function to return 0/1 if loading bitmap fails? If false then revert to `logo` ?